module2/assignment2.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Cook | INFO 2439 | Assignment 2</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="_css/Cook-a2.css">
+    <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+    <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+
+</head>
+
+<body>
+    <!-- Benjamin Cook
+     assignment?.html
+     INFO2439
+     Thoendel
+     9/22/2019
+-->
+    <div data-role="page" id="assignment2">
+        <div data-role="header" data-position="fixed">
+            <a href="index.html" data-icon="home" rel="external">Home</a>
+            <h1>Assignment #2</h1>
+        </div>
+        <h1>Restaurant Mobile Application Research</h1>
+        <div data-role="main" class="ui-content">
+            <ul data-role="listview" data-inset="true" rel="external">
+                <li><a href="#rmobileApp1">Taco Bell</a></li>
+                <li><a href="#rmobileApp2">McDonald's</a></li>
+                <li><a href="#rmobileApp3">Jimmy John's</a></li>
+            </ul>
+        </div>
+
+        <div data-role="footer" data-position="fixed" class="footer">Contact: <a href="mailto:bcook30@mail.mccneb.edu"
+                class="ui-btn-active">Benjamin Cook</a></div>
+    </div>
+
+    <div data-role="page" id="rmobileApp1">
+        <div data-role="header" data-position="fixed">
+            <a href="#assignment2" data-icon="back">Back</a>
```

Comment:
I would hope to see these back buttons be the dynamic back buttons from the chapters this week.